src/docfx/template/TemplateEngine.cs

```diff
@@ -51,7 +51,7 @@ public string Render(PageModel model, Document file)
 
         public JObject TransformMetadata(string schemaName, JObject metadata)
         {
-            return JObject.Parse(((JObject)_js.Run($"{schemaName}.mta.json.js", "transform", metadata)).Value<string>("content"));
+            return JObject.Parse(((JObject)_js.Run($"{schemaName}.mta.json.js".ToLowerInvariant(), "transform", metadata)).Value<string>("content"));
```

Comment:
This requires that the filename of files in the template folder are also in lowercase?

Reply:
right, let's just remove ToLower for now